docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md

```diff
@@ -56,7 +56,17 @@ For universes that use Google Cloud Provider (GCP), Amazon Web Services (AWS), o
 
 If you change the **Instance Type** or both the **Instance Type** and the **Volume Info** size and then click **Save**, YugabyteDB Anywhere gives you the option to either do a full migration of the universe and its data to new nodes, or do a smart resize, as per the following illustrations:
 
-  ![Full or smart resize1](/images/ee/edit-univ-1.png)
-  ![Full or smart resize2](/images/ee/edit-univ-2.png)
+![Full or smart resize1](/images/ee/edit-univ-1.png)
+![Full or smart resize2](/images/ee/edit-univ-2.png)
 
 If you change only the **Volume Info** size and click **Save**, YugabyteDB Anywhere automatically performs a smart resize.
+
+### Full Move
+
+The following changes in configurations will result in a full move
```

Comment:
Minor: It might be easier to describe what ops support a smart (@ddhodge has such a PR) that to list the opposite.